modules/vaos/spec/services/appointment_service_spec.rb

```diff
@@ -5,12 +5,55 @@
 describe VAOS::AppointmentService do
   subject { described_class.for_user(user) }
 
-  let(:user) { build(:user, :mhv) }
+  let(:user) { build(:user, :vaos) }
```

Comment:
The reason for the change from `mhv` to `vaos` isn't really clear -- there's no new factory and it seems like a lot of other places in the spec its a `mhv` user. Is there a consistent pattern for the kind of user in these specs?

Reply:
the rest will eventually use vaos... trying to keep this change incremental.